users/dierkes/preprocessing/wav2vec.py

```diff
@@ -60,8 +79,155 @@ def run(self):
                 frames = soundfile.info(fname).frames
                 dest = train_f if rand.random() > self.valid_percent else valid_f
                 print(
-                    "{}\t{}".format(os.path.relpath(file_path, dir_path), frames), 
-                        file=dest
+                    "{}\t{}".format(os.path.relpath(file_path, dir_path), frames),
+                    file=dest,
                 )
         if valid_f is not None:
             valid_f.close()
+
+
+class FairseqWav2VecModelConvertAndDumpJob(Job):
+    """
+    Conversion Job for fairseq model checkpoints to RETURNN model dictionary and checkpoint
+    """
+
+    def __init__(
+        self,
+        wav2vec_config,
+        output_name,
```

Comment:
Is it necessary to have this configurable? From my point of view, something generic like `"checkpoint"` would be fine.